gtests/net/mptcp/syscalls/client_selfconnect.pkt

```diff
@@ -0,0 +1,19 @@
+// connect() function, a client tries to connect to itself
+// ensure MPTCP fallback to TCP.
+--tolerance_usecs=100000
+`../common/defaults.sh`
+ 
+0      socket(..., SOCK_STREAM|SOCK_NONBLOCK, IPPROTO_MPTCP) = 3
++0     `nstat -n`
++0     connect(3, ..., ...) = -1 EINPROGRESS (Operation now in progress)
+ 
++0     >  S  0:0(0)                    <mss 1460, sackOK, TS val 100 ecr 0,         nop, wscale 8, mpcapable v1 flags[flag_h] nokey>
++0     <  S  0:0(0)         win 1000   <mss 1460, sackOK, TS val 407 ecr 0,         nop, wscale 8, mpcapable v1 flags[flag_h] nokey>
++0     >  S. 0:0(0)  ack 1             <mss 1460, sackOK, TS val 330 ecr 0,         nop, wscale 8, mpcapable v1 flags[flag_h] nokey>
++0     <  S. 0:0(0)  ack 1  win 65535  <mss 1460, sackOK, TS val 700 ecr 100,       nop, wscale 8, mpcapable v1 flags[flag_h] key[skey=2]>
++0     >   . 1:1(0)  ack 1             <nop,      nop,    TS val 845707014 ecr 700, nop, nop,      sack 0:1>
```

Comment:
why do you have such a huge value for the "TS val"? Can you not have something smaller in 3 digits, as long as it is bigger than 330?